phpBB/adm/style/admin.css

```diff
@@ -840,6 +840,7 @@ table.zebra-table tbody tr:nth-child(odd) {
 }
 
 .row2 {
+	word-break: break-all;
```

Comment:
Why not after the second property to keep in alphabetical classification?

Reply:
@Crizz0 We do no use alphabetical order for css we use font, design, layout order

Reply:
ah, okay.